extension/js/service_worker/bg-handlers.ts

```diff
@@ -149,8 +149,10 @@ export class BgHandlers {
   };
 
   public static thunderbirdContentScriptRegistration = async () => {
+    const contentScriptGroups = chrome.runtime.getManifest().content_scripts ?? []; // we know it's in the manifest
+    const files = contentScriptGroups[0].js?.map(url => url.replace(/moz-extension:\/\/[^/]+\//, './')) ?? [];
```

Comment:
This where dynamic content script injection comes in place. It's actually the best solution that I can't implement - its a great inclusion where we don't actually need to bundle the entire js libs and dependencies.